cw_bitcoin/lib/electrum_wallet.dart

```diff
@@ -617,7 +617,8 @@ abstract class ElectrumWalletBase extends WalletBase<ElectrumBalance,
       final transactions = await fetchTransactions();
       transactionHistory.addMany(transactions);
       walletAddresses.updateReceiveAddresses();
-      await transactionHistory.save();
+      final transactionDirection = transactions.map((key, value) => MapEntry(key, value.direction));
+      await transactionHistory.save(txsDirection:transactionDirection);
```

Comment:
shouldn't this be handled by default from `addOne` and `addMany` functions? they take the whole transaction object / transactions list

Reply:
@OmarHatem28 the direction of transactions in an existing wallet has been hardcoded as incoming,this property is overwritten in json in `save()` with the correct value.Had to call _load() twice before updating transactions from the server and after, since transactions from the device are loaded with the wrong direction